ports/sdbus-cpp/vcpkg.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "sdbus-cpp",
+  "version-string": "0.8.3",
```

```suggestion
  "version": "0.8.3",
```
This should be served by [version](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version).